### ISSUE TITLE
[dialog] Test sometimes locked on Python 2.7

### ISSUE THREAD

**Issue by vallsv** [silx-kit/silx#1669]
Must be investigated

This test never ending (time to time). The process must be killed manually. Maybe due to an infinite while on tests.

```
python run_tests.py --installed -vv silx.gui.dialog.test.test_imagefiledialog.TestImageFileDialogInteraction.testClickOnBackToRootTool
```

**Comment by vallsv** [comment]
Well, still deadlock sometimes...

**Comment by vallsv** [comment]
I am afraid `QTest.qWaitForWindowShown` (Qt4) is not using a timeout, while `QTest.qWaitForWindowExposed` (Qt5) is using a timeout. The Qt4 version wait for event... No?

**Comment by vallsv** [comment]
That's maybe still a problem with network drivers... Boring...

**Comment by vallsv** [comment]
That's it. It is deadlocking before `qWaitForWindowExposedAndActivate`... Maybe because of network drivers...

**Comment by vallsv** [comment]
Problem found and fixed for the release 0.7. An instance of `QFileDialog` was created to custom user shortcuts. It is now disabled on Windows PyQt4.
